src/Microsoft.ML.FastTree/FastTree.cs

```diff
@@ -3404,4 +3377,47 @@ public TreeNode(Dictionary<string, object> keyValues)
             public Dictionary<string, object> KeyValues { get; }
         }
     }
+    internal static class FastTreeIniFormatUtils
```

Comment:
Can you add this as a separate file in `FastTree/Utils`. I'm not sure we need it in `FastTree.cs`

Reply:
Moved it to utils

---
In reply to: [243139101](https://github.com/dotnet/machinelearning/pull/1929#discussion_r243139101) [](ancestors = 243139101)